src/Symfony/Component/DependencyInjection/Compiler/AutowirePass.php

```diff
@@ -619,4 +629,20 @@ private function populateCombinedAliases(ContainerBuilder $container): void
             }
         }
     }
+
+    private static function resolveAutowireAttribute(\ReflectionAttribute $attribute, \ReflectionClass $class): mixed
+    {
+        if (self::$autowireAttributeResolver) {
+            return (self::$autowireAttributeResolver)($attribute, $class);
+        }
+
+        self::$autowireAttributeResolver = function (\ReflectionAttribute $attribute, \ReflectionClass $class) {
```

Comment:
the previous "if" can be removed and this be changed to `self::$autowireAttributeResolver ??=`